ansible/roles/baselayout/vars/main.yml

```diff
@@ -104,7 +104,7 @@ packages: {
   ],
 
   rhel8: [
-    'ccache,cmake,gcc-c++,gcc-toolset-10,gcc-toolset-10-libatomic-devel,gcc-toolset-11,gcc-toolset-12,gcc-toolset-13,git,make,python3',
+    'ccache,cmake,gcc-c++,gcc-toolset-10,gcc-toolset-10-libatomic-devel,gcc-toolset-11,gcc-toolset-12,gcc-toolset-12-libatomic-devel,gcc-toolset-13-libatomic-devel,git,make,python3',
```

Comment:
I guess we don't need `gcc-toolset-13` explicitly because `gcc-toolset-13-libatomic-devel` depends on it?

Reply:
ah that's an oversight. I'd rather be specific -- I'll add it back.

Reply:
Add back explicit gcc-toolset-13.